idaes/models_extra/power_generation/costing/power_plant_costing.py

```diff
@@ -96,13 +232,239 @@
                 "Project Contingency": 0.2,
                 "RP Value": 493587.88,
                 "Units": "lb/hr",
+            },
+            "6.1.ccs":{
+                    "BEC": 6128000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 1074.424688,
+                    "Units": "m**3"
+            },
+            "6.2.ccs":{
+                    "BEC": 6040000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 791.6813487,
+                    "Units": "m**3"
+            },
+            "6.3.ccs":{
+                    "BEC": 1986000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 419.6971436,
+                    "Units": "m**3"
+            },
+            "6.4.ccs":{
+                    "BEC": 1438000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 309.2505268,
+                    "Units": "m**3"
+            },
+            "6.5.ccs":{
+                    "BEC": 260000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 800,
+                    "Units": "m**2"
+            },
+            "6.6.ccs":{
+                    "BEC": 3095000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 4250,
+                    "Units": "m**2"
+            },
+            "6.7.ccs":{
+                    "BEC": 1151000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 9100,
+                    "Units": "m**2"
+            },
+            "6.8.ccs":{
+                    "BEC": 465000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": 1200,
+                    "Units": "m**2"
+            },
+            "6.9.ccs":{
+                    "BEC": 731000/1e3,
+                    "Eng Fee": 0.2,
+                    "Process Contingency": 0.12,
+                    "Project Contingency": 0.2,
+                    "RP Value": [262349.78, 672504.35],
+                    "Cost scaling fraction": [0.6, 0.4],
```

Comment:
We do not need to add cost scaling fractions. 
EPAT typically uses two accounts for example 5.1.a and 5.1.b. 
Lets keep the method consistent with existing data base.